pos_receipt_hide_user/static/src/xml/order_receipt.xml

```diff
@@ -0,0 +1,14 @@
+<templates id="template" xml:space="preserve">
+    <t t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension" owl="1">
+        <xpath expr="//div[hasclass('cashier')]" position="attributes">
+                <attribute name="t-att-class">{'oe_hidden': true}</attribute>
```

Comment:
When HideUserOption is empty, the name of cashier is still hidden, i think it should be shown as normal.

Reply:
Hi anh @nilshamerlinck and anh @nguyenminhchien, so it means:

- User is allow to show employee name on receipt?
- If not, I agree HideUserOption should not be empty. I will try to make it either 'remove completely' or 'replace by triagram'